### ISSUE TITLE
Adding New Categories Fails

### ISSUE THREAD

**Issue by DyoungMCWI** [WordPress/gutenberg#10774]
WordPress 4.9.8 running Boxy theme.
Gutenburg Version 4.0.0
Chrome Version 70.0.3538.67 (Official Build) (64-bit)

Trying to add a new Catagory to a post w/Gutenburg. When clicking the "Add new category" link, nothing appears to happen.  This was not the case prior to updating the plugin to 4.0.0.

**Comment by ajitbohra** [comment]
Kindly share the browser console output?

**Comment by arunsathiya** [comment]
Noting that I tried to reproduce this with the same specifics, and I am not able to reproduce this. Clicking on the `Add new category` link works as expected. 

<img width="1120" alt="screenshot 2018-10-19 at 21 42 45" src="https://user-images.githubusercontent.com/18581859/47230481-f1a79d00-d3e7-11e8-9d87-1af84570b6ad.png">
<sup>Image indicating that a new category was created</sup> 

--- 

Tested on a local install of WordPress. WordPress 4.9.8 and Gutenberg 4.0.0 using Google Chrome Version 70.0.3538.67 (Official Build) (64-bit) on macOS 10.14 - Mac OS Mojave
